findimports.py

```diff
@@ -38,6 +38,8 @@
   -c, --collapse        collapse dependency cycles
   -T, --tests           collapse packages named 'tests' and 'ftests' with
                         parent packages
+  -s MODULE, --search-import MODULE
+                        only show imports matching module or package name
```

Comment:
So basically if there's a line of code that does

    from a.b import c as d

then it will be shown when I --search for `a` or `b` or `a.b`, but I'm not sure about `c` (or `a.b.c`)?  And AFAIU it won't be found if you --search for `d`?  Is that correct?

Reply:
Correct, it will be able to find `a`, `b`, `c`, `a.b` and even `b.c` and `a.b.c` while not having a clue about `d`.